src/webgpu/api/operation/texture_view/format_reinterpretation.spec.ts

```diff
@@ -0,0 +1,68 @@
+export const description = `
+Test texture views can reinterpret the format of the original texture.
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import {
+  kRenderableColorTextureFormats,
+  kRegularTextureFormats,
+} from '../../../capability_info.js';
+import { GPUTest } from '../../../gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+function supportsReinterpretation(a: GPUTextureFormat, b: GPUTextureFormat) {
+  return a + '-srgb' === b || b + '-srgb' === a;
+}
+
+g.test('texture_binding')
+  .desc(`Test that a regular texture allocated as 'format' may be sampled as 'viewFormat'.`)
```

Comment:
nit: since these are operation tests, "may be" should be something like "are correctly"

Reply:
> For all valid 'format'/'viewFormat' pairs, test that any regular texture allocated as 'format' is correctly sampled as 'viewFormat'.

similar below